### PR TITLE
basic files done #6

### PR DESCRIPTION
**Pull request by alMutazBeAllah**
basic files done #6

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    
+</body>
+</html>>
```
Comment (mhmdtshref) [inline public/index.html]: remove the unneeded `>`

**Comment by mattlub** [comment]
.gitignore?

package.json?

**Comment by ameentalahmeh** [comment]
> .gitignore?
> 
> package.json?

**Comment by alMutazBeAllah** [comment]
@mattlub  

> .gitignore?
> 
> package.json?

Mohammed & Amin did is on anther branch

**Pull request merged**